internal/pkg/manifest/lb_web_svc.go

```diff
@@ -103,12 +69,58 @@ type HTTPHealthCheckArgs struct {
 }
 
 // HealthCheckArgsOrString is a custom type which supports unmarshaling yaml which
-// can either be of type string or type HealthCheckArgs. q
+// can either be of type string or type HealthCheckArgs.
 type HealthCheckArgsOrString struct {
 	HealthCheckPath *string
 	HealthCheckArgs HTTPHealthCheckArgs
 }
 
+// HTTPHealthCheckOpts converts the ALB health check configuration into a format parsable by the templates pkg.
+func (hc HealthCheckArgsOrString) HTTPHealthCheckOpts() template.HTTPHealthCheckOpts {
+	opts := template.HTTPHealthCheckOpts{
+		HealthCheckPath:    defaultHealthCheckPath,
+		HealthyThreshold:   hc.HealthCheckArgs.HealthyThreshold,
+		UnhealthyThreshold: hc.HealthCheckArgs.UnhealthyThreshold,
```

Comment:
Is there a reason why these two fields are set directly? Instead of checking if they are not nil and then assigning them

Reply:
it's because they are both pointer values. The reason why path is not doing it this way is because we want to set default value for path. And there exists some type conversion for interval and timeout if they are not nil.

Reply:
derp gotcha yea that makes sense